kpi/deployment_backends/base_backend.py

```diff
@@ -32,6 +32,16 @@ class BaseDeploymentBackend(abc.ABC):
     Defines the interface for a deployment backend.
     """
 
+    PROTECTED_XML_FIELDS = [
+        '__version__',
+        'formhub',
+        'meta',
+    ]
+
+    # XPaths are relative to the root node
+    SUBMISSION_UUID_XPATH = './meta/deprecatedID'
+    FORM_UUID_XPATH = './formhub/uuid'
+
```

Comment:
For future references, this PR removes the double slashes from XPaths previously set. 

@jnm said:
> My understanding is that .// will match any descendant, while ./ looks for an immediate child